intro-HPC/intro-HPC.tex

```diff
@@ -76,8 +76,7 @@ \part{Advanced Guide}
 %TODO Write this chapter for each site!
 %\inputsite{ch_monitoring_utilization}
 
-%TODO: write this chapter!
-%\include{ch_checkpointing}
+\include{ch_checkpointing}
```

Comment:
@redfast00 Probably not for everyone, only for UGent & VUB (@wpoely86?)

Reply:
VUB, yes. I thought UA too? @sbecuwe ?

Reply:
Runs were not always stable, so we no longer advertise it to our users.